matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/tasks/RedactEventTask.kt

```diff
@@ -26,22 +28,31 @@ internal interface RedactEventTask : Task<RedactEventTask.Params, String> {
             val txID: String,
             val roomId: String,
             val eventId: String,
-            val reason: String?
+            val reason: String?,
+            val withRelations: List<String>?
```

Comment:
Maybe we should add a trailing comma here?

Reply:
https://github.com/vector-im/element-android/pull/7988/commits/300e167577729e9e9ad7abce43dad25e2f80aa76